app/reducers/channels.js

```diff
@@ -386,6 +387,15 @@ const filtersSelector = state => state.channels.filters
 const filterSelector = state => state.channels.filter
 const nodesSelector = state => state.network.nodes
 
+const displayNodeName = (channel, nodes) => {
+  const node = find(nodes, n => channel.remote_pubkey === n.pub_key)
+
+  if (node && node.alias.length) { return node.alias }
+  return channel.remote_pubkey ? channel.remote_pubkey.substring(0, 10) : channel.remote_node_pub.substring(0, 10)
+}
+
+const caseInsensitiveIncludes = (left = '', right = '') => left.toLowerCase().includes(right.toLowerCase())
```

Comment:
I would inline this and DRY up another way - e.g. maybe call `filterChannel`/`filterPendingChannel` within the `channelArray` filter loop?